getting-started-tutorial/01-mlrun-basics.ipynb

```diff
@@ -4,242 +4,254 @@
-    "* Artifact path - In order to store the artifacts you need to set the artifact_path to the desired root folder of your artifacts. <br>\n",
-    "Artifacts from each run are stored in the artifact_path which can be set globally through environment var (MLRUN_ARTIFACT_PATH) or through the config, if its not already set we can create a directory and use it in our runs.  <br>\n",
-    "Using {{run.uid}} in the path will allow us to create a unique directory per run. <br>\n",
-    "You can use {{run.project}} to include the project name in the path. <br>\n",
-    "Later on when you run your jobs you can use {{run.uid}} to include the specific run uid in the artifact path. <br>\n",
-    "if you don't do that then the jobs artifact will overwrite the old ones. <br>\n",
-    "if you want to store a new artifact every time you run your job then use run.uid <br>"
+    "<a id=\"gs-tutorial-1-mlrun-envr-init\"></a>"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Pre-requisite"
+    "### Initializing Your MLRun Environment\n",
+    "\n",
+    "Define a project name and run the `set_environment` MLRun method and set the `project` parameter to your project name to initialize the MLRun development environment for your project.\n",
+    "You can optionally pass additional parameters to `set_environment`, such as `artifact_path` to override the default path for storing project artifacts, as explained in the next steps.\n",
+    "`set_environment` returns the current artifacts path."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "If you already installed MLRun then ignore this step. if not, then run the script below <br>\n",
-    "The script run pip install mlrun with the mlrun version that is aligned to the mlrun service running in the platform"
+    "#### Defining the Project Name"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 3,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Both server & client are aligned (0.6.0rc9).\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "!/User/align_mlrun.sh"
+    "# Set the project name\n",
+    "project_name = '-'.join(filter(None, ['getting-started-tutorial', getenv('V3IO_USERNAME', None)])).lower()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "> **Note:**  Restart jupyter kernel after running the above script"
+    "#### Configuring the Artifacts Path\n",
+    "\n",
+    "You can configure a default MLRun artifacts path.\n",
+    "In the platform, the default artifacts path is a **&lt;project name&gt;/artifacts** directory in the predefined \"projects\" data container &mdash; **/v3io/projects/&lt;project name&gt;/artifacts** (for example, **/v3io/projects/myproject/artifacts** for a \"myproject\" project).\n",
```

Comment:
comment about v3io artifact path should be made as a note, specific to running on iguazio (not in open source)